src/android/MixpanelPlugin.java

```diff
@@ -229,7 +229,7 @@ private boolean handlePeopleSetOnce(JSONArray args, final CallbackContext cbCtx)
 
     private boolean handlePeopleSetPushId(JSONArray args, final CallbackContext cbCtx) {
         String pushId = args.optString(0);
-        mixpanel.getPeople().setPushRegistrationId(pushId);
+        mixpanel.getPeople().setPushRegistrationId('"'+pushId+'"');
```

Comment:
yup it needs to be built:
https://github.com/mixpanel/mixpanel-android/blob/v4.7.0/BUILDING.txt